Application/Frontend/components/Navbar/notifcation.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect, FC } from 'react';
+import { Menu, Divider, Text, Notification, Button } from '@mantine/core';
+
+interface FriendRequest {
+  id: string;
+  username: string;
+}
+
+interface InboxDropdownProps {
+  userId: string;
+}
+
+const InboxDropdown: FC<InboxDropdownProps> = ({ userId }) => {
+  const [sentRequests, setSentRequests] = useState<FriendRequest[]>([]);
+  const [receivedRequests, setReceivedRequests] = useState<FriendRequest[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+
+  const fetchFriendRequests = async () => {
+    setLoading(true);
+    try {
+      const sentResponse = await fetch('/api/Get-SentFriendRequest', {
```

Comment:
Again, please revise your API endpoint names to follow our naming standard. This endpoint should be renamed to `get-sent-friend-requests`